cleanse/cleanse.go

```diff
@@ -529,6 +550,13 @@ func toObjects(records []map[string]string, accepts acceptsFunction, f convertFu
 	return sortObjects(added)
 }
 
+func LoadCountryDuties() []CountryDuty {
```

Comment:
This is fine - but given we will have one default per country, can also return this as a `map[string]CountryDuty` which will make lookup simpler